test/pytest/receiver/plugins/test_asapo_receiver.py

```diff
@@ -30,7 +30,7 @@ def config():
 
 @pytest.fixture
 def worker(config):
-    config = {k: v for k, v in config.items() if k != "user_config_path"}
+    del config["user_config_path"]
```

Comment:
A copy would be safer. I don't know what happens if a test uses both, the `worker` and the `plugin` fixture, together.

Reply:
Done.